packages/cli-e2e/utils/login.ts

```diff
@@ -8,14 +10,20 @@ function isLoginPage(page: Page) {
   return page.url().match(/https:\/\/platform.*cloud\.coveo\.com\/login/);
 }
 
-function isLoggedIn(page: Page) {
-  return page.url().match(/http:\/\/127\.0\.0\.1:32111/);
+export async function isLoggedin() {
+  const currentAccount = userInfo().username;
+  const accessToken = await getPassword(
+    'com.coveo.cli.access.token',
+    currentAccount
+  );
+  return accessToken !== null;
```

Comment:
The `isLoggedin` function is now based on the keychain rather than a redirection to the redirect uri

Reply:
I would widen the check to falsy value
```suggestion
  return Boolean(accessToken);
```

I know you can use `!!` to do the same thing, but I think it's clearer because explicit.